app/react/V2/Routes/Settings/CustomUploads/components/UploadProgress.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Translate } from 'app/I18N';
+import { useRecoilValue } from 'recoil';
+import { uploadProgressAtom } from './uploadProgressAtom';
+
+type UploadProgressProps = {
+  queueLength: number;
+};
+
+const UploadProgress = ({ queueLength }: UploadProgressProps) => {
+  const { filename, progress } = useRecoilValue(uploadProgressAtom);
+
+  return filename ? (
+    <div className="flex grow">
+      <Translate>Uploading</Translate>...&nbsp;
+      <span className="font-semibold">{filename}</span>&nbsp;
+      {progress}%&nbsp;
+      <span className="flex flex-nowrap before:content-['('] after:content-[')']">
```

Comment:
Why not just add the "(" to the HTML?

Reply:
They don’t convey meaning, it’s just for looks. In the past for this type of case we have done this to allow users who customize the UI to decide if they want the parenthesis or not, since they are there for looks only. They would not have to alter the HTML to remove the parentheses, just touch the custom CSS. I don’t have a preference one way or another.